duckduckgo/Android
# Default browser: swap buttons order + update copy

Task/Issue URL: https://app.asana.com/0/488551667048375/1199369070928655/f
Tech Design URL: 
CC: 

**Description**:
This PR applies the changes proposed in https://app.asana.com/0/715106103902962/1145186821227282/f
- update copy
- swap buttons order
- update secondary button style

See product review: https://app.asana.com/0/1142021229838617/1199369070928657/f

**Steps to test this PR**:
1. Fresh install
1. When in Default browser screen
1. ensure buttons are laid out as expected and copy is updated ("Set as Default Browser")


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
